docs/examples/Agilent_N9000A.ipynb

```diff
@@ -0,0 +1,148 @@
+{
+ "metadata": {
+  "nbsphinx": {
+   "execute": "never"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.7-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python_defaultSpec_1598349077504",
+   "display_name": "Python 3.7.7 64-bit ('python37': conda)"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example with the Agilent N9000A"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import qcodes as qc\n",
+    "import sys\n",
+    "sys.path.append('../../')\n",
```

Comment:
Please don't use path modifying tricks in the documentation. qcodes_contrib_drivers is a package that can be installed normally. E.g. `pip install .` or `pip install -e .` from the root folder of the repository (or when this makes it into a release `pip install qcodes_contrib_drivers`

Reply:
I modified the import.